packages/block-editor/src/components/inner-blocks/index.native.js

```diff
@@ -73,9 +73,13 @@ function UncontrolledInnerBlocks( props ) {
 	const {
 		clientId,
 		allowedBlocks,
+		prioritizedInserterBlocks,
+		__experimentalDefaultBlock,
+		__experimentalDirectInsert,
 		template,
 		templateLock,
 		templateInsertUpdatesSelection,
+		__experimentalCaptureToolbars: captureToolbars,
```

Comment:
Do we _need_ these ones?

Reply:
I don't understand this question.

Reply:
The question is "why are we adding all these extra props when this PR is about `prioritizedInserterBlocks`"?